### DISCUSSION TITLE
Using bard-api I made bard-prompt; a command-line tool to manage, log, etc.

### DISCUSSION THREAD

**Discussion by jaggzh** [dsdanielpark/Bard-API#83] [Show and tell]
Actually, nevermind. I think the less attention there is to the api the less of a potential problem it might be for google.
I am open to hearing your ideas on this, so I'll leave this as a topic for discussion then.

**Comment by dsdanielpark** [comment]
First and foremost, I introduced the project directly to the official Google Cloud community and inquired about the official project's release date. I also wrote a post, as requested by Google, stating that I am willing to abandon or make the project private at any time.

Furthermore, with unofficial APIs like Bard-API, it is possible to develop various approaches using macros or requests. Additionally, Google can easily control how frequently users use different prompts and what prompts they are using through targeted cookies tied to user accounts, and Google can respond to this easily at any time.

However, similar to companies working on chat GPT and other LLM services, Google likely determines service development priorities based on the usage patterns and frequencies of various prompts, and uses user behavior as backdata to improve Bard. This is essential and valuable data in catching up with pioneers like OpenAI's chat GPT.

Therefore, if Google collects sufficient backdata and decides to sell a paid version rather than this experimental version of Bard, they can quickly control and manage these aspects within seconds.

Hence, my project's level of interest and sustainability are not significantly relevant.

I am researching and studying LLM to make it usable and perform inference with minimal resources using various methods. I plan to disclose the ongoing project soon.

This project simply aims to make it easier for diverse developers to re-implement and replace it when the official Bard API is released, with the ultimate goal of developing a democratic open-source LLM that can replace Bard.
